.github/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 5. Enhancement - Changed default coastline colour - thanks to @SamLefevre (Samuel Lefevre)
 6. Enhancement - Updated Edinburgh (EGPH) FAVA colour - thanks to @SamLefevre (Samuel Lefevre)
 7. Enhancement - Updated Glasgow (EGPF) FAVA colour - thanks to @SamLefevre (Samuel Lefevre)
+8. AIRAC (2306) - Updated Lerwick/Tingwall (EGET) Runway Headings - thanks to @rishab-alt (Rishab Saddiq(Rish))
```

Comment:
Hi @rishab-alt, just regarding the real name in brackets at the end of the changlog entry, adding an extra name in brackets inside it means the parser we use on the changelog at release time will not catch it as a name and you won't get any credit! So probably best to stick to whatever your prefered name is, e.g `(Rish Saddiq)` or `(Rishab Saddiq)` or just `(Rish)` or whatever.